rest_framework_social_oauth2/oauth2_grants.py

```diff
@@ -90,6 +91,11 @@ def validate_token_request(self, request):
 
         try:
             user = backend.do_auth(access_token=request.token)
+            user_data = backend.user_data(access_token=request.token)
+            exp = user_data['exp']
+            if exp is not None and exp <= int(time.time()):
```

Comment:
from datetime import datetime
this expression can be improved to this:
```suggestion
            if not exp and exp <= datetime.now():
```

considering that exp has a datetime in it